rush-plugins/rush-azure-storage-build-cache-plugin/src/schemas/azure-interactive-auth.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Configuration for Azure interactive auth prompt",
+
+  "type": "object",
+
+  "additionalProperties": false,
+
+  "required": ["storageAccountName", "storageContainerName"],
+
+  "properties": {
+    "storageAccountName": {
+      "type": "string",
+      "description": "(Required) The name of the the Azure storage account to use for build cache."
+    },
+
+    "storageContainerName": {
+      "type": "string",
+      "description": "(Required) The name of the container in the Azure storage account to use for build cache."
+    },
+
+    "azureEnvironment": {
+      "type": "string",
+      "description": "The Azure environment the storage account exists in. Defaults to AzurePublicCloud.",
+      "enum": ["AzurePublicCloud", "AzureChina", "AzureGermany", "AzureGovernment"]
+    },
```

Comment:
Shouldn't these just come from the `build-cache.json` config file?

Reply:
These are a separate config because the purpose of this plugin is to be able to reuse the existing Azure auth stack for non-build-cache tasks that need to authenticate to Azure, such as deploying assets to a storage account for testing.